### ISSUE TITLE
Reality Shock: Video "Memories of a bartender"

### ISSUE THREAD

**Issue by micheleminno** [humanetech-community/humanetech-community-awareness#11]
# "Memories of a bartender" Deliverable

## Info 

- Name: **Memories of a bartender**
- Type: **Video**
- Campaign: [Reality shock](campaigns/reality-shock/README.md)
- General discussion: [Forum topic title here](forum-topic-url)

## Summary 

Short video about the evolution of social relations in a shared space (a bar).

## Goals

- Offer a glimpse of how we used to relate each other in the past (from the 50's onwards, to now). Especially valuable for the younger generations, who never experienced anything of that.

- let the viewer get the impact of the situation now. Done by contrast, after having seen how we used to relate during the past decades. 

## Concept

Each generation has their own language, style, music and - most of all - their own ways of getting in touch with each other and express themselves. In private as well as in public spaces. Through the eye of a neutral observer - the bartender - we show the different generations expressing themselves, taking actions and having fun in a bar. At the end the actual situation - silent people smiling, gazing, waiting at their phones - will result in an environment that gradually gets ever more strange and creepy. 

The message posed to the viewer is "Watch how we used to have meaningful, funny, troublesome, creative interactions and how these changed over time. All this ended up like this today, cooling down to people sitting one near the other but far on their screens. If you are one of them (we are all in a way or another one of them) you can feel now all the emptiness and sadness of this actual condition".

## Strategy

- This deliverable does not require additional strategy. Please refer to [campaign strategy](campaigns/reality-shock/README.md#strategy).

## Script

Mockumentary-style video about the incredible story of a bartender who witnessed the passing of decades (social, cultural music changes) since ‘50s.

The whole scene takes place in a bar, ideally located in Europe/US main west world trendsetter cities (New York, San Francisco, London, ..). An old bartender (75 years old or so) speaks about his memories at his work place, that bar, where he worked since the 50's and now he owns too. He speaks to an interviewer on the other side of the bar that we never see. He would ideally look like [him](https://www.google.it/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=2ahUKEwiI3qbBw_7dAhVKsqQKHYSLAy0QjRx6BAgBEAU&url=https%3A%2F%2Fwww.imdb.com%2Ftitle%2Ftt0758758%2Fmediaviewer%2Frm868764672&psig=AOvVaw39ydQhVWbKk95wy3mKt7qm&ust=1539269275830727).

#### Initial scene 
* Music: Closer, The Chainsmokers ft. Halsey  (soft music, like that one played in supermarkets or waiting rooms, enjoyable but not too disturbing).

We see the face of the bartender at his side of the bar (we don't see the front bar, only the back, with bottles etc.), who says: "I'm here since 1956, this place has been my life".

We see some old pictures at the wall behind him. Camera zooms into the first one ('50s one) and then scene passes to memories of the bartender. The still image comes to life and turns into the scene. Each time a scene ends, camera zooms out and shift to another picture (black/white for '50s and 60s, color for the others) and zooms in. Scenes are always color scene also for '50s and '60s.

#### Memories scenes
Scenes of the bar during the passing of decades, starting from the '50s. Each time a year is superimposed at the bottom of the screen. There's no stage in the bar so there's no scene with a band playing live. Also because it would be a cross-decades situation and not characteristic of a specific time.

##### '50s scene 
* The bottom line says: "1958". 
* Music: Rumble, by Link Wray. 

A boy laughing and surrounded by his friends is pushed, blindfolded, to go to all woman in the bar asking for a date, because he had lost at a game of pool.

##### '60s scene 
* The bottom line says: "1968". 
* Music: White room, by Cream.

Boys and girls are animately discussing (we can't hear their voice), with flayers, posters about some protest activity on the streets they are planning.

##### '70s scene 
* The bottom line says: "1977". 
* Music: How Deep Is Your Love, Bee Gees.

People is warmly dancing in couples of all kinds (man-girl, man-man, girl-girl, ..), with all extravagant fashion and hairstyles.

##### '80s scene
* The bottom line says: "1987". 
* Music: Welcome to the jungle, Guns N' Roses.

A fight among people in the bar, with broken glasses, chairs, etc.

##### '90s scene
* The bottom line says: "1994". 
* Music: Loser, Beck.

One girl is singing the song at the karaoke, followed by most of the other people in the bar, singing the chorus together (grunge clothes: flannel shirts, jeans, [cheap clothes with no specific care](https://www.wikihow.com/Dress-Grunge)). Someone else is playing at arcade videogames, surrounded by small groups watching and commenting loud.

##### '00s scene
* The bottom line says: "2003". 
* Music: Drum beat from "8 mile" movie

There's a rap battle going on between two guys and all public inciting them.

#### Present times scene
* Music: Closer, The Chainsmokers ft. Halsey

The face of the bartender comes back (same music as at the beginning). He says with a sad face: "Now it's all so silent and creepy." And after two seconds he shakes his head and says: "It's time for me to sell this place". The camera broadens, flips slowly and we see the bar now, full with silent people, each one seated and looking down on their phone screen. It all looks strange and creepy (sounds of tapping on screens). Screen black with white writing:"Are you one of them?".

##### Insights about old bartender's feelings 
The old man feels the sadness but with words he tries to justify the situation. Maybe he doesn’t understand the new situation because he’s old (he thinks naively). He feels peace but also sadness, that’s why he want to quit. He is not against technology, in every decade he used technology to attract customers and make them feel good. I.e. karaoke in the '90s, arcade videogames, etc.. It must be evident by the images that all these technology generate sociality among customers, instead of repress it. At present time he's doing all it's needed to please customers (soft music, wifi, recharge points, ..) and he should be happy to have satisfied clients, silence and peace. But nevertheless we can see he's sad: there's a conflict inside himself between owner's goals (to maximize customers and keep the bar clean and safe) and human being's goals (live among people who relate to each other, socialise, express themselves). So he decides to quit. It is too much, and the place is no longer the place he loved to work in for decades.

#### Final scene
* Music: Tired boy, Joey Pecoraro

We see the same bar, with even more aseptic forniture (all white) and music (i.e. electronic beats), with people seated on their phones and a new young bartender, a young woman, herself on the phone, silent.
Music continues during credits on black screen.

### Scenography

Each decade has its own furniture, games, lights, etc.: 

* present: wifi signal, recharge points for phones.
* '50s: pool tables, jukebox
* '60s: posters on the walls, flags, smoke.
* '70s: strobe lights
* '80s: 
* '90s: arcade videogames and karaoke
* '00s:
* final scene: white walls

(see first part of Black Mirror episode 'San Juniper' for inspiration).

## Duration

About 2 minutes:

* The initial scene is about 10 seconds long.
* Each decade scene is between 10 and 15 seconds long.
* The present time scene is about 10 seconds long.
* The final scene is about 5 seconds long.

## Music

* present time scene: [Closer, The Chainsmokers ft. Halsey](https://www.youtube.com/watch?v=PT2_F-1esPk). 
* '50s: [Rumble, Link Wray](https://www.youtube.com/watch?v=ucTg6rZJCu4).
* '60s:  [White room by Cream](https://www.youtube.com/watch?v=VR90gQ-SIaY).
* '70s: [How deep is your love, Bee Gees](https://www.youtube.com/watch?v=XpqqjU7u5Yc).
* '80s: [Welcome to the jungle, Guns N' Roses](https://www.youtube.com/watch?v=o1tj2zJ2Wvg).
* '90s: [Loser, Beck](https://www.youtube.com/watch?v=YgSPaXgAdzE).
* '00s: [Drum beat from "8 mile" movie](https://www.youtube.com/watch?v=oVb0jvnHEpQ).
* final scene: [Tired boy, Joey Pecoraro](https://www.youtube.com/watch?v=F7khiq25Xr0):

## Production

We would need to find a professional video producer. They will have to propose and arrange a shooting venue (the bar) and secure actors.

Otherwise we should have to find different bars with the same structure, at least similar (i.e. same placement of the windows). In this way we can shoot a scene in a '80s bar in London, another in a '50s bar in New York and so on. Viewers would automatically think it's the same bar with different forniture.
 

## Funding

- Funding requirements should still be determined.

## Team

- To be determined

## Resources

- This deliverable has no additional resources. Please edit or comment to add your relevant links.

## Notes about this version
* Voice of the bartender removed during the remembered scenes.
* The bartender is also the owner of the bar.
* Past scenes are introduced by the corresponding picture on the wall.

**Comment by georgeshk** [comment]
Love the idea of a natural constant observer (the bar owner) and a constant place (the popular bar), witnessing transformations through generations. First, it is mostly music (am I wrong?) and tastes, but suddenly it is everything (the omnipresence of the phone). So (again, am I wrong?), it would depict a gentle evolution, followed by a shocking new paradigm: the phones changed everything!

**Comment by micheleminno** [comment]
Thank you, yes the phones changed everything, uniforming the actions and reactions of people, pushing people to internalize more and more feelings and thoughts. Before, there was music, jokes, noise, discussions, even fights, after only clean well educated silent people, each one seated and looking down on their screen, not expressing or showing nothing at the exterior, in that moment, in that place.

**Comment by aschrijver** [comment]
(@micheleminno I have updated your issue to comply to the template I just created, with minor edits)

I love :heart: your idea too. It inspired me for the [Video deliverable proposal](.github/ISSUE_TEMPLATE/video-deliverable-proposal.md) template sample text. But I gave it a different twist. Allow me provide some feedback..

- **Potential reach**: Having a spoken narrative in the video narrows the audience to people that understand that language (I presume English here). You can make language versions or use subtitles (in Youtube this doesn't  involve creating different video versions), but without spoken text it would be easier for a global audience.
 
- **Funding**:  I agree, a professionally-made video will quickly need some level of funding. But I think we want to go too professional too quick. We need to get a whole organization in place before we get there. On the other hand I think - in this case - amateurs can create a professional looking video in a crowdsourced manner. More on this below.

- **Cost**: Now that I've addressed funding, let's talk about cost to produce this video. We can optimize the script to minimize cost. Couple of things come to mind:

  - No spoken language: No professional actors needed, only figurants (amateurs)
  - Select the kind of bar that never changed during the decades (e.g. like an Irish pub). No shifting of furniture needed (just increasing nr. of pictures / memories on the wall)
  - Amateur equipment nowadays give good quality results (look e.g. what many vloggers are doing). Think of a 'steady-cam' device for low-cost video / photo camera. We'll find people who have this.

Now I will introduce a variation of the script (a copy from the issue template, actually). If you like it, we can adopt changes, otherwise create 2 scripts and chose later, or use both.

### Script alternative

The video starts in the 50's, with a steady-cam shot going through the door of a lively bar. There is a lively atmosphere and many guests. The camera goes straight to the barman, who smiles broadly while cleaning a glass.

In the next shot, we see the barman from the back and the year 1958 appears. The video speeds up, and we see the years pass by in a couple of seconds, showing quick flashes of people in different settings in the bar. The video slows again at 1968, where we see a wedding party with people dancing to live music from a band in the corner, uproarous laughter.

[...] This goes on, speeding up to the next decade, slow down and show a new scene. [...]

Now when the video slows we are in 2018. The bar is mostly silent, while it is still filled with people. On every table we see smartphones, people showing their screen to the person next to them, not aware of others in the bar. We still see the barman from the back. He is trying to draw attention from a guest whose drink is waiting on the counter. Eventually he has to raise his voice. The viewer can deduce that he has lost some luster and joy in his work.

Now the years click by slowly one by one, with sped up video fragments showing the same quiet people, phone zombies really. There are less people too. We end in 2021 when suddenly the bar is empty, furniture is covered in sheets. The bar is forever closed. The barman has quit, retired. This was not his calling.

The camera that was behind the counter all the time now slowly pans into the bar, going along the walls where many photo frames have been hung over the years. They show the good moments of the past, The Days of Old. When reading the door, the steady-cam slowly goes outside, only to nearly avoid collision to a young passer-by who is absorbed in his phone. The person mumbles something ineligible and walks on. The street outside is empty, save for one person walking with AR glasses on her face, playing some game.

No slowly the large text "Do you really want this?" fades into view. The camera pans down, showing the pavement and a hand appears holding a smartphone. The smartphone has a screen only showing a big 'Off' button. The hand's thumb presses it, and with a blimp the video goes to black. CHT logo + URL appears followed by credit roll.

---

Notes:

- I like the last part, which focuses specifically on the smartphone, after the _"environment gets gradually ever more strange and creepy"_, so the message is clearer, still without spoken text.

- There are ways to make this cheaper and easier to film: Instead of sped up movement in the bar, there can be just fade ins and fade outs between the decades, while the year counter is going to the present day

I have an idea for crowdsourcing all this and actually create a bunch of video's (or at least lay a solid basis for them) in one go. But that is about strategy, and I will create a separate issue for it.

**Comment by georgeshk** [comment]
@aschrijver

Fantastic, very thorough feedback. I hope Michele you will like it too. This is getting closer to a detailed storyboard.

On production: as you know, I am very much inclined to go for professionals. You have demonstrated great talent for putting things into place and addressing many technical aspects. However, I am concerned with the special art of production, including arranging shooting venues, directing figurants, and coming up with the right cinematic effects. If we seek to produce videos in-house, we may end up with an amateurish feel, that will put off the audience.

**Comment by micheleminno** [comment]
Hi @aschrijver , thank you for the editing of the template. 

I read your alternative version, I like it but I think it gives a different flavour, more like science fiction. Instead I wanted to create the atmosphere of plain reality, of a sort of mockumentary around the character od the old bartender, and without any final dystopian future scenario. I want to keep it light in terms of persuasion, just showing the reality and a man of another generation - the bartender - leaving the scene because he doesn't fit anymore with that new landscape. In this way, we present the viewer with a choice, and give them more responsibility to feel the bads and goods of the alternatives without any strong judgement. It's clear that the whole idea is that the bar with silent people, each one closed within themselves is bad, the viewers get this by themselves by the sad feeling of the old man and the creepy sounds of background music and tapping on screens.

About removing the spoken language, I agree, I though about it too. We can remove the voice telling memories during the passing decades (they can be directly depicted in the scenes). We can leave something at the beginning and at the end in a very simple english, something like:
* "I'm here since 1952, this place has been my life" (at the beginning)
* "Now it's all silent and peaceful, it's time for me to leave this job" (at the end)

**Comment by aschrijver** [comment]
> I like it but I think it gives a different flavour, more like science fiction

That is no problem. We could have 2 scripts, though we still seem reasonably aligned. There is no science fiction, or dystopic effect in the script I propose.

More a melancholy when the end reflects back to the beginning, and there is the realization that "something" is really lost. In both script versions the barman retires, though in mine the bar is closed (probably you found that dystopic).

Your minimization of speech to just beginning and end phrases is a good idea. But it is "Now it's all silent and peaceful, it's time for me to leave this job" doesn't seem to me to get the message across. It's like 'All is well and good now.. the new generation is different and that is fine'. Maybe that is your intention, though.

> Maybe crowdsourcing each unit of video representing a different decade?

This aligns with the idea I wanted to post a new issue for: Organizing an **Awareness Hackathon** across al our Meetup groups. This would also fit well with the _"shoot each decade in a different bar"_ idea of yours, but I think that would weaken the impact of the video.

**Comment by micheleminno** [comment]
# But the "Now it's all silent and peaceful, it's time for me to leave this job" sentence doesn't seem to me to get the message across. It's like 'All is well and good now.. the new generation is different and that is fine'. Maybe that is your intention, though.

No, it’s not my intention, how bad is the situation will result by contrast. The man feels the sadness but with words he tries to justify the situation. He feels peace but also sadness, that’s why he want to quit. 

Let’s see, we can do both and wait for other comments/contributions..

Yes you’re right, there’s no science fiction or dystopia, just a little..

**Comment by georgeshk** [comment]
Yes the fact that the bartender is old should be handled with caution (older people more resistant to technology, therefore more likely biased). Remember we should in this  campaign make observations, but remain as neutral as we can, and leave it to the audience to reach their own conclusions and go into introspection.

I like the idea that the bar ends up close and the camera looking at pictures of happy moments spanning decades. Happy moments gone, soon the memories themselves will be gone too when the bar is eventually dismantled.

**Comment by georgeshk** [comment]
One more general comment on the Reality Shock campaign and each video initiative. Each is the "baby" of the person who first proposed a concept for a new video, and as such, this person should take the lead in all aspects of making it come to realization. At the same time, all concept originators such as you @micheleminno and others such as yours truly should be involved in shaping up other video concepts that are not their own. This will help address common issues and cross-pollineate different deliverable ideas.

One note of caution though: we need to respect one another's creativity and not challenge a new video idea in its core principles (which are laid out when the deliverable issue is created), but rather assist in accomplishing the stated objectives, improving the script, and realizing the intended effect.

**Comment by micheleminno** [comment]
> Maybe show the bartender using a phone at some point, but very quickly, say to text his wife "Be home soon" and that is it. This could alleviate concerns about general bias towards technology in any form.

Good point, he is not against technology because we can see that in every decade he uses technology to attract customers and make them feel good. I.e. karaoke in the '90s, arcade videogames, etc. it must be evident by the images that all these technology generate sociality among customers, instead of repress it. At present time he's doing all it's needed to please customers (soft music, wifi, recharge points, ..) and he should be happy to have satisfied clients, silence and peace. But nevertheless we can see he's sad: there's a conflict inside himself between owner's goals (to maximize customers and keep the bar clean and safe) and human being's goals (live among people who relate to each other, socialise, express themselves). So he decides to quit.

**Comment by georgeshk** [comment]
Yes, that sounds very meaningful to me. He has gotten accustomed to all evolutions of technology through generations, but this one, he cannot take it anymore. It is too much, and the place is no longer the place he loved to work in for decades.

His quitting or closing the bar (I like to think he is the owner) means an end to an era. Viewers sympathize with a old bar owner, because he has seen a lot, knows better than most, and may be the wise referential.

As I said earlier, I like the idea of the whole place shut down with furniture covered in drapes and the camera spanning old pictures of past happy moments.

**Comment by georgeshk** [comment]
The idea of a new bartender is good (I was thinking of a new, modern business taking over the location to highlight the transition).

It represents a new generation taking over from the old, but that is all uniform and character-less.

This is the scary prospect for many of us who have witnessed the whole evolution: in our old age, will everything be unlike what we used to love?

**Comment by georgeshk** [comment]
@micheleminno 

There has been quite a lot of feedback and suggestions on your idea. Would you care to update/expand the script contained in your first post, taking on board all such comments that we all seem to agree on?

(As discussed in my previous post, concept originators ideally should take the lead on developing their initiative, maintaining/updating all relevant details such as scripts, etc. Once the detailed script is finalized by the group, we can move the deliverable description in the "Code" section, and move on to discuss production-related issues)

**Comment by micheleminno** [comment]
Yes, I will do it. I was also starting to arranging a first draft of the readme file, with some pictures useful to convey better my idea.. Is it better to update the issue or write the updated version directly on the draft readme file (or both)?

**Comment by georgeshk** [comment]
> Is it better to update the issue or write the updated version directly on the draft readme file (or both)?

I would propose for you to update the first post at the top of this issue (and add a comment at the bottom explaining the changes you made in a few words). My thinking is the concept designer maintains the script, hence no need to keep all earlier versions, otherwise makes up for top much reading.

**Comment by georgeshk** [comment]
Also @micheleminno

Please start thinking of all production options we discussed so far (crowdsourced, visual arts academies, or professionals) and indicate which of these avenues you would propose.

@patmasu and @aschrijver suggest visual arts academies, I think that is a good compromise between crowdsourced and professional production. Unless we are ok to wait for a long time, I dont believe we can crowdsource production. If you like it, how about giving some thoughts to visual arts academies in your area? I can produce a template email/letter but you may have to translate!

**Comment by georgeshk** [comment]
@micheleminno 

I guess you made changes to the script on top of this issue, but not sure which (I was thinking you could explain key changes in a post).

One comment on the script: maybe considering cutting the number of eras that are visited, lest it becomes too repetitive (also may be an issue that each scene would be too short).

Would be great if you could add to your script with music/songs proposals for each era, as well as details of what happens in each scene. One possible idea I had was a good old' bar fight to the sound of "Welcome to the jungle" by Guns n' Roses.

I have not re-read all posts above in detail, but I believe @aschrijver had more very good suggestions you could take on board. Personally, I am not that much in favor of the barman retiring, I'd prefer to think he is the owner (well, he takes care of serving customers, but the bar is his own, i.e. his identity) and that he eventually closes the place down. For your consideration and that of other contributors here.

**Comment by micheleminno** [comment]
> I guess you made changes to the script on top of this issue, but not sure which (I was thinking you could explain key changes in a post).

I'm adding the key changes in the last part of the issue

> maybe considering cutting the number of eras that are visited, lest it becomes too repetitive (also may be an issue that each scene would be too short).

Let's see when all decades have a story and music, maybe they are too many but it could be useful and enjoyable (if each one is very characteristic and different from other ones), mainly for youngers who didn't experience anything of them.

> Would be great if you could add to your script with music/songs proposals for each era, as well as details of what happens in each scene.

Yes I'm working on it.

> One possible idea I had was a good old' bar fight to the sound of "Welcome to the jungle" by Guns n' Roses.

Good idea, I'll add it in the '80 decade.

>  Personally, I am not that much in favor of the barman retiring, I'd prefer to think he is the owner (well, he takes care of serving customers, but the bar is his own, i.e. his identity) and that he eventually closes the place down. 

According to me it's better to keep the bar alive. If we shut it down, it means there's ho hope for socialization anymore. Remember that the bar at present time is full of people, so even if they are at sociality level 0.1, they still want to be physically near each other (even if on their phones). 
Another practical point is that a commercial activity closes if it has no customers anymore, but this is not the case. The old bartender (who can be the owner too) doesn't leave for economic reasons.

**Comment by georgeshk** [comment]
> Another practical point is that a commercial activity closes if it has no customers anymore, but this is not the case. 

Hi @micheleminno

Good point. Maybe he sells the bar? Problem with old bartender is it would be unclear if he retires of old age or for other reasons (which he would need to explain in the voice-over). Anyway, not a highly critical point, just believe it is stronger if he is the owner (a bartender employee working all his life in the same bar for someone is unlikely).

**Comment by micheleminno** [comment]
Sure, he's the owner and he sells the bar. The reason of his choice is clear by his sad face and what he says (I'm changing that a little bit to make clear it's not because of his old age).

**Comment by georgeshk** [comment]
> but it could be useful and enjoyable (if each one is very characteristic and different from other ones), mainly for youngers who didn't experience anything of them.

Good point again. Once all scenes are proposed, we can all discuss which ones are not so rich/interesting, if any, and skip.

Re. the technologies he adopts in each era, not a problem if we cannot think of one per era (that could prove difficult). As long as a few key ones appear (as you suggested, arcade games, karaoke, etc), that would work.

**Comment by georgeshk** [comment]
Hi @micheleminno 

Was checking on this project and realized you have done a lot to elaborate on the script. Remember to let others know when you do this so we can tune in (we don't get alerts when you edit a post, only if you post a new comment).

It is great, now quite detailed. Any comment on the script, anyone?

As suggested before, take some time to think about and propose your strategy for making this video concept a reality, i.e. how to produce it, and how to finance production if need be. 

My thinking: very hard to shoot the video on our own. Very hard. A lot of logistics involved. Not only finding people, attires for each period, but also furniture, the bar's entire decoration changes (or maybe doesn't have to) throughout the years. I tend to think we would need to find a very professional, and well-equipped videographer for this one.

**Comment by georgeshk** [comment]
@micheleminno 

Was just thinking. Instead of the bar owner talking to himself about why he is sad, we could have a more natural way to deal with this. Let us say we have a scene near the end where he discusses the sale of his bar the new owner. The new potential owner would ask "I love this place, and would be happy to make you a very good offer. But I am curious. The place is doing fine, why would you sell it?" Then the bar owner gives his reasons and misgivings about the recent trends. That would be much more natural. What do you think?

**Comment by micheleminno** [comment]
> Let us say we have a scene near the end where he discusses the sale of his bar to a new owner. The new potential owner would ask "I love this place, and would be happy to make you a very good offer. But I am curious. The place is doing fine, why would you sell it?" Then the bar owner gives his reasons and misgivings about the recent trends.

My idea was that the old man speaks to an invisible person who is interviewing him (like in a documentary or mockumentary), so his speaking is actually answers to this interviewer, that we don't see.

**Comment by georgeshk** [comment]
I am thinking about production issues. We would need to find different bars that look similar, but have a different vibe.

Much easier to shoot all scenes in the same bar (through all eras, the place looks the same, meaning nothing drastic has happened).

**Comment by georgeshk** [comment]
> My idea was that the old man speaks to an invisible person who is interviewing him

Got it, but please give more thoughts to my suggestion. Somebody wants to buy the bar so would have a good reason to chat the bar owner. Why would the bar owner be interviewed?

**Comment by micheleminno** [comment]
> Why would the bar owner be interviewed?

Because of his incredible memories from '50s to today, being a witness to all cultural, society, music changes through the years.

**Comment by georgeshk** [comment]
Agreed, but a business dealing conversation would look more natural than an interview. He could say the exact same things. 

Up to you to decide what's best.

**Comment by micheleminno** [comment]
> Agreed, but a business dealing conversation would look more natural than an interview. He could say the exact same things.

I prefer my version, so last scene with the new bartender at the phone has more impact. It gives a feeling of a documentary too, that can be underlined with proper choices about the film colors, video editing, etc.

**Comment by georgeshk** [comment]
> I prefer my version, so last scene with the new bartender at the phone has more impact.

All of us are merely offering our opinions and suggestions. I feel that every concept originator - you, in this instance - should be the final judge of that input. We will support your vision.

**Comment by micheleminno** [comment]
Sure, thank you for your suggestions, they make my idea sharper and bring me to analyse things I had overlooked.

**Comment by aschrijver** [comment]
This proposal has come to a very complete stage. Great work @micheleminno and @georgeshk !
If all the comments have been incorporated, this can be brought to a document at: `/campaigns/reality-shock/video/memories-of-a-bartender/README.md`

**Comment by Dori-CHT** [comment]
I agree, great work!   I have a suggestion for another song for 1987:  Whitney Houston's "I Wanna Dance w/Somebody".  Maybe it's because I never liked Guns & Roses, but I think it could work to create a better impression of that era (BPZ - before phone zombies).  Yes, more dancing....

**Comment by micheleminno** [comment]
Thank you @Dori-CHT! Guns 'n Roses serve to the fight scene in the bar, and to break with '70s scene, also dancing (Bee Gees)..

**Comment by Dori-CHT** [comment]
Guns'n Roses does serve to mix it up - can't be a phone zombie in the middle of a bar brawl!  The music in general serves very well to remind people of the time before smartphones took over.

**Comment by micheleminno** [comment]
I updated the readme in the moab (memories of a bartender) folder.

**Comment by micheleminno** [comment]
Next time I’ll reference and close the issue directly by the commit.

**Comment by aschrijver** [comment]
Note: If on your commit text you add the issue ID, then the commit is referenced in the issue e.g. `updated doc XYZ (#11)`
